src/components/ClipboardButton.ts

```diff
@@ -1,20 +1,26 @@
 import { delay } from '@terra-ui-packages/core-utils';
 import * as clipboard from 'clipboard-polyfill/text';
 import _ from 'lodash/fp';
-import { useState } from 'react';
+import { PropsWithChildren, ReactNode, useState } from 'react';
 import { h } from 'react-hyperscript-helpers';
-import { Link } from 'src/components/common';
+import { Link, LinkProps } from 'src/components/common';
 import { icon } from 'src/components/icons';
 import { withErrorReporting } from 'src/libs/error';
 import * as Utils from 'src/libs/utils';
 
-export const ClipboardButton = ({ text, onClick, children, iconSize = undefined, ...props }) => {
+interface ClipboardButtonProps extends LinkProps {
```

Comment:
Suggest including `children` in the props interface.

```suggestion
interface ClipboardButtonProps extends PropsWithChildren<LinkProps> {
```

Same comment for other components.